HiP-Achievements/Controllers/ActionBaseController.cs

```diff
@@ -31,11 +32,12 @@ public ActionBaseController(EventStoreService eventStore, InMemoryCache cache)
             _entityIndex = cache.Index<EntityIndex>();
         }
 
+        [Obsolete("Use UserStore instead")]
```

Comment:
This attribute should only be in the ExhibitVisistedController not in the base class